src/app/modules/shared/components/details-fields/details-fields.component.html

```diff
@@ -118,6 +120,8 @@
             aria-describedby="inputGroup-sizing-sm"
             [class.is-invalid]="end_date.invalid && end_date.touched"
             required
+            [max]="getActualDate()"
+            onkeydown="return false"
```

Comment:
Only a question, what does this line do?

Reply:
this line does not allow you to write the dates you can only select with the calendar

Reply:
because if that line doesn't go you could write future dates